_locales/en/messages.json

```
@@ -242,14 +242,14 @@
   "extensionHasUpdated": {
     "message": "Scratch Addons has updated to v$1"
   },
-  "extensionUpdateInfo1_v1_31": {
-    "message": "The \"insert blocks by name\" addon was completely reworked. You can now type inputs and reporters inside other blocks, and autocomplete using the Tab key. Go to $1 to enable it."
+  "extensionUpdateInfo1_v1_32": {
+    "message": "New addon: \"move sprite to front layer\", which lets you Shift+Click a sprite within the sprite pane to move it to the front (top layer) of the stage. Go to $1 to enable it."
```

Maybe we could try something else instead of "new addon:"?
```suggestion
    "message": "In this release the main new addon is \"move sprite to front layer\", which lets you Shift+Click a sprite within the sprite pane to move it to the front (top layer) of the stage. Go to $1 to enable it."
```

---

This syntax is also used since 2021 but it's never too late to change it.

---

We should also make sure the notification is easy and fast to read.

---

Let's stick with "New addon:" for this release.